docs/main/updating/plugins/4-0.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Updating plugins to 4.0
+description: Guide for updating a Capacitor v3 plugin to v4
+slug: /updating/plugins/4-0
+---
+
+# Updating a Capacitor v3 plugin to 4.0
+
+There are only a few changes required to update a plugin from Capacitor 3 to Capacitor 4.
+
+:::info
+While you can follow the manual steps below, check out [this tool](https://github.com/rdlabo-team/capacitor-plugin-to-v4) from community member [@rdlabo](https://twitter.com/rdlabo) that automatically updates a Capacitor 3 plugin to v4.
+:::
+
+## Android
```

Comment:
I followed [this commit](https://github.com/ionic-team/create-capacitor-plugin/commit/03027bf603626ebfdba084cb9d9a1a359d008554#diff-a875f0630e8a46d5a0e706a94502051f5a75e669312987096e4f52941b2bf476) to figure out what changed. I may have missed something. if it's easier, maybe we just recommend the migration tool?